.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": [
+    "eslint-config-airbnb",
+    "plugin:prettier/recommended"
+  ],
+  "parser": "babel-eslint",
+  "root": true,
+  "env": {
+    "browser": true,
+    "es6": true
+  },
+  "rules": {
+    "no-param-reassign": [0],
+    "no-underscore-dangle": [0],
+    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }]
```

Comment:
what's the advantage of doing this?

Reply:
You can put JSX in .js files.

Reply:
Should we stick with one way? At work we use JS, because well, that's what is in there :)

Reply:
I'm fine with that, but I think all this stuff needs moving over to a separate issue to discuss how/where we deviate from CRA